### ISSUE TITLE
Add information about cost of deployment

### ISSUE THREAD

**Issue by rishijavia** [hybox/aws#171]
Add the cost for initial setup in README file. I know these services are elastic and can scale up but we can calculate the basic cost of deployment when the initial servers/services are created.

Would give perspective to departments who are under budget constraint.

**Comment by bbranan** [comment]
I agree that it would be useful to list the AWS resources which are created by these templates and their associated prices. Until this is done, you may find the table listed here to be helpful: https://wiki.duraspace.org/display/hyku/Hyku+Product+Beta+-+Frequently+Asked+Questions
